### DISCUSSION TITLE
More slots on toolbar?

### DISCUSSION THREAD

**Discussion by spectangulation** [mriscoc/Ender3V2S1#811] [Ideas]
Is it possible to have additional slots on the toolbar? It has quickly become one of my favourite parts of this firmware, but I want to keep what is present and add more quick access options.

**Comment by mriscoc** [comment]
It Is possible to add more options from where to choose in the toolbar_def.h file.
About the number of shortcuts in the toolbar, maybe 5 items are enough. It could be possible to change the amount of them in TBMaxOpt from the dwin_defines.h file.

**Reply by spectangulation** [reply]
can i do this myself with VS?

**Comment by JEBurroughs** [comment]
427 board using the Ender3V2-427-BLTUBL-MPC-20230805 bin and I don't get a visible toolbar. It wont save anything changes I make either. I save and if I reboot, it's all back to default. The toolbar is invisible. If I scroll to it, I don't see it but I can click on it. I have the Dwin display. Do I need to update my screen?

**Reply by mriscoc** [reply]
https://github.com/mriscoc/Ender3V2S1/issues/1012#issuecomment-1667295446

**Comment by JEBurroughs** [comment]
I re-flashed the screen with the current DWIN and same thing.

I just loaded the manual mesh bin and I see the toolbar now. I have to experiment some more.

**Comment by JEBurroughs** [comment]
WOW, I loaded the Ender3V2-427-BLTUBL-MPC-20230805 again and same issue with toolbar being invisible. The Manual Mesh version seemed to look and save normally. For some reason the BLT version doesn't want to work correctly.

**Reply by mriscoc** [reply]
https://github.com/mriscoc/Ender3V2S1/issues/1012#issuecomment-1667295446
